CONTRIBUTING.md

```diff
@@ -0,0 +1,19 @@
+### Introduction
+
+Thank you for showing interest for contributing in our new FOSS United Platform which we are building for the FOSS Community all over India. You can always take a look at what are the current issues we are working on. There are a lot of ways you can contribute to our platform including requesting Documentation, Bug Reports or Feature Requests and help us make our platform efficient and better at performance
```

Comment:
nitpicky comment - for prose, the editors we use are usually designed around handling long lines (line wrapping by default). With code editors on the other hand, line wrapping usually isn't enabled by default so we might have to manually horizontally scroll the file to read the contents.

If we set a line length limit (e.g. 100 characters) and manually wrap the lines, that will improve readability and editability in the code editors.

See for example the README file in the CPython repository https://raw.githubusercontent.com/python/cpython/main/README.rst. It's an ReST syntax file but still, the general guidance around setting a line length limit still applies.